examples/mbedTLS/CMakeLists.txt

```diff
@@ -0,0 +1,72 @@
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-mbedtls)
+
+hunter_add_package(mbedtls)
+find_library(
```

Comment:
> Do you want me to hunterize this package inside hunter_packages and add CMake code to generate mbedtlsConfig.cmake ?

Yes, I think users of `mbedtls` will be grateful. Also it's not about Hunter so can be contributed upstream.

I've created fork:
* https://github.com/hunter-packages/mbedtls

But I don't know with what branch to start.